dds_web/api/project.py

```diff
@@ -588,10 +607,15 @@ def post(self):
 
         try:
             db.session.commit()
-        except (sqlalchemy.exc.SQLAlchemyError, TypeError) as err:
+        except (sqlalchemy.exc.SQLAlchemyError, sqlalchemy.exc.OperationalError, TypeError) as err:
             flask.current_app.logger.exception(err)
             db.session.rollback()
-            raise DatabaseError(message="Server Error: Project was not created") from err
+            raise DatabaseError(
+                message=str(err),
+                alt_message="Server Error: Project was not created" + (": Database malfunction.")
+                if isinstance(err, sqlalchemy.exc.OperationalError)
+                else ".",
+            ) from err
```

Comment:
```suggestion
            raise DatabaseError(
                message=str(err),
                alt_message=(
                    "Project was not created" 
                    + (": Database malfunction." 
                        if isinstance(err, sqlalchemy.exc.OperationalError)
                        else "."),
                 )
            ) from err
```
Aand same here with the linting 😁